index.js

```diff
@@ -138,10 +138,11 @@ export class UpdateAPK {
       console.log("RNUpdateAPK::getAppStoreVersion - iosAppId doesn't exist.");
       return;
     }
-    const URL =
-      "https://itunes.apple.com/us/app/apple-store/id" +
-      this.options.iosAppId +
-      "?mt=8";
+    //const URL =
+    //  "https://itunes.apple.com/us/app/apple-store/id" +
+    //  this.options.iosAppId +
+    //  "?mt=8";
```

Comment:
I don't know a lot about the URL format, but I'm curious if you know why the `?mt=8` is no longer needed?

Reply:
I just tested it with one of my ids and seems to work fine, :+1: